backend/btrixcloud/orgs.py

```diff
@@ -583,6 +584,17 @@ async def update_event_webhook_urls(
         )
         return res is not None
 
+    async def update_crawling_defaults(
+        self, org: Organization, defaults: CrawlConfigDefaults
+    ):
+        """Update crawling defaults"""
+        res = await self.orgs.find_one_and_update(
+            {"_id": org.id},
+            {"$set": {"crawlingDefaults": defaults.dict()}},
```

Comment:
`dict` is deprecated, `model_dump` is what's recommended instead
```suggestion
            {"$set": {"crawlingDefaults": defaults.model_dump()}},
```